diff --git a/node_modules/y-monaco/dist/y-monaco.js b/node_modules/y-monaco/dist/y-monaco.js
index 0261cb7..98918a4 100644
--- a/node_modules/y-monaco/dist/y-monaco.js
+++ b/node_modules/y-monaco/dist/y-monaco.js
@@ -160,7 +160,9 @@ class MonacoBinding {
       this._rerenderDecorations();
     };
     ytext.observe(this._ytextObserver);
-    monacoModel.setValue(ytext.toString());
+    if (monacoModel.getValue() !== ytext.toString()) {
+      monacoModel.setValue(ytext.toString())
+    }
     this._monacoChangeHandler = monacoModel.onDidChangeContent(event => {
       // apply changes from right to left
       this.mux(() => {
diff --git a/node_modules/y-monaco/src/y-monaco.js b/node_modules/y-monaco/src/y-monaco.js
index 182786f..2f2823c 100644
--- a/node_modules/y-monaco/src/y-monaco.js
+++ b/node_modules/y-monaco/src/y-monaco.js
@@ -100,18 +100,18 @@ export class MonacoBinding {
                 if (anchorAbs.index < headAbs.index) {
                   start = monacoModel.getPositionAt(anchorAbs.index)
                   end = monacoModel.getPositionAt(headAbs.index)
-                  afterContentClassName = 'yRemoteSelectionHead'
+                  afterContentClassName = 'yRemoteSelectionHead yRemoteSelectionHead-' + clientID
                   beforeContentClassName = null
                 } else {
                   start = monacoModel.getPositionAt(headAbs.index)
                   end = monacoModel.getPositionAt(anchorAbs.index)
                   afterContentClassName = null
-                  beforeContentClassName = 'yRemoteSelectionHead'
+                  beforeContentClassName = 'yRemoteSelectionHead yRemoteSelectionHead-' + clientID
                 }
                 newDecorations.push({
                   range: new monaco.Range(start.lineNumber, start.column, end.lineNumber, end.column),
                   options: {
-                    className: 'yRemoteSelection',
+                    className: 'yRemoteSelection yRemoteSelection-' + clientID,
                     afterContentClassName,
                     beforeContentClassName
                   }
@@ -156,7 +156,9 @@ export class MonacoBinding {
       this._rerenderDecorations()
     }
     ytext.observe(this._ytextObserver)
-    monacoModel.setValue(ytext.toString())
+    if (monacoModel.getValue() !== ytext.toString()) {
+      monacoModel.setValue(ytext.toString())
+    }
     this._monacoChangeHandler = monacoModel.onDidChangeContent(event => {
       // apply changes from right to left
       this.mux(() => {
@@ -202,7 +204,7 @@ export class MonacoBinding {
     this._monacoChangeHandler.dispose()
     this.ytext.unobserve(this._ytextObserver)
     this.doc.off('beforeAllTransactions', this._beforeTransaction)
-    if (this.awareness !== null) {
+    if (this.awareness) {
       this.awareness.off('change', this._rerenderDecorations)
     }
   }
